docs/source/howto/lcf.rst

```diff
@@ -0,0 +1,467 @@
+
+Data Acquisition
+-----------------------------------
+
+This workflow requires spatially explicit Earth Observation raster data to start with. It will technically work with any multi- or hyperspectral image data. This tutorial has been created with spectral-temporal metrics aggregated from all clear-sky Sentinel-2 observations in 2018 over Berlin, Germany (MGRS Tile 33UUU).
+
+We first download all image acquisitions with less than 70% cloud cover according to the metadata catalogue using FORCE Level 1 Cloud Storage Downloader:
+
+.. code-block:: bash
+
+   force-level1-csd -s S2A,S2B -d 20180101,20181231 -c 0,70 /path/to/metadata/directory/ /path/to/datapool / /path/to/datapool/pool.txt T33UUU
+
+We do not provide these data in the downloadable data because of file size, and as they can be downloaded again anytime using the above command.
+
+Tip
+  
+Please refer to the Level 1 Cloud Storage Downloader `Documentation <https://force-eo.readthedocs.io/en/latest/components/lower-level/level1/level1-csd.html>`_ and `Tutorial <https://force-eo.readthedocs.io/en/latest/howto/level1-csd.html>`_ to know more about Sentinel-2 data downloads and about retrieving and updating the metadata catalogue required to download raw image data.
+
+
+Data Pre-Processing
+-----------------------------------
+
+FORCE provides all functionalities to convert all downloaded Level 1 data (i.e., radiometrically calibrated and georectified) into Analysis-Ready Data (ARD), or Level 2 data. ARD are data that are readily usable for any application without much further processing. In FORCE, this includes 
+
+- cloud and cloud shadow detection,
+- radiometric, atmospheric, and topographic correction,
+- BRDF correction,
+- resolution merging (from 20m bands to 10m, Sentinel-2 only).
+
+For processing, we use
+
+.. code-block:: bash
+
+   force-level2 /path/to/parametertile/directory/10_lcf_level_2.prm
+
+You can access the parameter file `here <../_static/parameter-files/tutorials/lcf/10_lcf_level_2.prm>`_ or use the one provided in the data repository. 	
+
+It is highly recommended to use a Digital Elevation Model (DEM) for topographic correction purposes. We here use a global SRTM/ASTER composite that we cannot provide for download. However, you can use any DEM of your choice here, for example the one provided by the `Copernicus Land Monitoring Service <https://www.eea.europa.eu/data-and-maps/data/copernicus-land-monitoring-service-eu-dem>`_. The use of a DEM is, however, not required to continue data processing, and as our example region is rather flat, the impact of missing topographic correction might be acceptable.
```

Comment:
mention global Copernicus DEM, and point to DEM tutorial? https://force-eo.readthedocs.io/en/latest/howto/dem.html